src/pages/Widget/Preview.tsx

```diff
@@ -11,9 +13,12 @@ export default function Preview({ classes = "" }) {
 
   return (
     <section className={`${classes} @container/preview pb-4`}>
-      <h2 className="text-lg @4xl/widget:text-2xl text-center @4xl/widget:text-left mb-3">
-        That's what our Donation Form looks like:
-      </h2>
+      <span className="max-sm:hidden">
```

Comment:
```suggestion
      <div className="max-sm:hidden">
```
nit: inline element `span` shoudn't contain block element `p`

Reply:
Fixed in 07587398.